stl/inc/xstring

```diff
@@ -363,14 +364,7 @@ public:
         _In_reads_(_Count) const _Elem* const _First2, const size_t _Count) noexcept /* strengthened */ {
         // compare [_First1, _First1 + _Count) with [_First2, ...)
 #if _HAS_CXX17
-#if _HAS_U8_INTRINSICS
-        if constexpr (is_same_v<_Elem, char8_t>) {
-            return __builtin_u8memcmp(_First1, _First2, _Count);
-        } else
-#endif // _HAS_U8_INTRINSICS
-        {
-            return __builtin_memcmp(_First1, _First2, _Count);
-        }
+        return __builtin_memcmp(_First1, _First2, _Count);
 #else // _HAS_CXX17

```

Comment:
Why not remove the preprocessor conditionals and use `__builtin_memcmp` unconditionally?

Reply:
Doesn't compile in C++14 mode. From the Microsoft-internal build:
```
S:\msvc\binaries\x86chk\inc\xstring(366,16): error C3861: '__builtin_memcmp': identifier not found [S:\msvc\src\vctools\winrt\vcmetadatareader\vcmetadatareader.nativeproj]
S:\msvc\src\vctools\crt\github\stl\inc\xstring(366,16): error C3861: '__builtin_memcmp': identifier not found [S:\msvc\src\vctools\crt\managed\xmd\msvcurt_netcore\msvcurt.nativeproj]
```

Reply:
That's silly: MSVC-PR-250606.